instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/metrics/DurationHistogram.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.metrics;
```

Comment:
Can you place it in a `...metrics.internal` package? I don't think we want this to become a part of the public  `instrumentation-api` API